utils/docker/cloud/.env

```diff
@@ -0,0 +1,23 @@
+# Copyright 2021-2022 Intel Corporation
+# All rights reserved.
+#
+# Configuration file for DAOS docker compose
+
+# XXX Arguments which must be defined XXX
+# DAOS_CLIENT_UNAME="foo"
+# DAOS_CLIENT_UID="666"
+# DAOS_CLIENT_GNAME="bar"
+# DAOS_CLIENT_GID="999"
+
+# XXX Arguments which must be updated if needed XXX
+DAOS_AGENT_RUNTIME_DIR="/var/run/daos_agent"
+
+# Arguments which should not be updated
+DAOS_DOCKER_IMAGE_TAG="rocky8.6"
+RHEL_BASE_IMAGE_NAME="rockylinux/rockylinux"
+RHEL_BASE_IMAGE_TAG="8.6"
+BUTST_CACHE=""
```

Comment:
Is it a typo here? BUST_CACHE="" ?

Reply:
Oups, indeed a typo error => going to fix it.